validator/lib/validateTileset.js

```diff
@@ -95,31 +97,40 @@ function validateTileHierarchy(root, tilesetDirectory) {
         }
 
         if (defined(content) && defined(content.boundingVolume)) {
-            var contentRegion = content.boundingVolume.region;
-            var contentSphere = content.boundingVolume.sphere;
-            var contentBox = content.boundingVolume.box;
-            var tileRegion = tile.boundingVolume.region;
-            var tileSphere = tile.boundingVolume.sphere;
-            var tileBox = tile.boundingVolume.box;
-
-            if (defined(contentRegion) && defined(tileRegion) && !regionInsideRegion(contentRegion, tileRegion)) {
-                return 'content region [' + contentRegion + '] is not within tile region + [' + tileRegion + ']';
+            var contentBV = content.boundingVolume;
+            var tileBV = tile.boundingVolume;
+            var tileTransform = Matrix4.IDENTITY;
+            if(defined(tile.transform)) {
+                tileTransform = Matrix4.fromArray(tile.transform);
             }
-
-            if (defined(contentSphere) && defined(tileSphere) && !sphereInsideSphere(contentSphere, tileSphere)) {
-                return 'content sphere [' + contentSphere + '] is not within tile sphere + [' + tileSphere + ']';
+            var contentTransform = Matrix4.IDENTITY;
+            if(defined(content.transform)) {
+                contentTransform = Matrix4.fromArray(content.transform);
             }
+            var message = undefined;
 
-            if (defined(contentBox) && defined(tileBox) && !boxInsideBox(contentBox, tileBox)) {
-                return 'content box [' + contentBox + '] is not within tile box [' + tileBox + ']';
+            message = checkBoundingVolume(contentBV, tileBV, contentTransform, tileTransform);
```

Comment:
```
var message = undefined;

message = checkBoundingVolume(contentBV, tileBV, contentTransform, tileTransform);
```

Can be `var message = checkBoundingVolume(contentBV, tileBV, contentTransform, tileTransform);`